src/app/georeferencer/qgsgeorefmainwindow.cpp

```diff
@@ -1098,6 +1101,23 @@ void QgsGeoreferencerMainWindow::createMapCanvas()
 
   mCentralLayout->addWidget( mCanvas, 0, 0, 2, 1 );
 
+  // snapping + cad
+  mAdvancedDigitizingDockWidget = new QgsAdvancedDigitizingDockWidget( mCanvas );
+  addDockWidget( Qt::LeftDockWidgetArea, mAdvancedDigitizingDockWidget );
+  mAdvancedDigitizingDockWidget->hide();
+  connect( mActionAdvancedDigitizingDock, &QAction::triggered, mAdvancedDigitizingDockWidget, [=]( bool checked ) { mAdvancedDigitizingDockWidget->setVisible( checked ); } );
+
+  QgsSnappingConfig snappingConfig;
+  snappingConfig.setMode( Qgis::SnappingMode::AllLayers );
+  snappingConfig.setTypeFlag( Qgis::SnappingType::Vertex );
+  snappingConfig.setTolerance( 10 );
```

Comment:
Shouldn't you use the default snaping tolerance?

Reply:
I don't find any default snapping tolerance!

Reply:
Should be ["default-snapping-tolerance"]( https://github.com/qgis/QGIS/blob/9059f200a081b163ed8793fa44eb904f480d31c9/resources/qgis_global_settings.ini#L28) available in Options->Map Tools->Digitizing->Snapping